CMakeLists.txt

```diff
@@ -10,6 +10,7 @@ project(Tetris)
 file(GLOB srcMain Source/*.cpp)
 file(GLOB srcScreens Source/Screens/*.cpp)
 file(GLOB srcButtons Source/Buttons/*.cpp)
+file(GLOB srcSettings Source/Settings/*.cpp)
```

Comment:
Why separate globs? You can add all the source files like this:
`
file(GLOB src
   Source/*.cpp)
   Source/Screens/*.cpp)
   Source/Buttons/*.cpp)
   Source/Settings/*.cpp
)
`
Or even simplier:
`
file(GLOB_RECURSE src
   Source/*.cpp)
)
`
BUT! It's easy, but it's not recommended way to find source files like it's said in docs([cmake org](https://cmake.org/cmake/help/v2.8.11/cmake.html#command:file)):

> GLOB will generate a list of all files that match the globbing expressions and store it into the variable. Globbing expressions are similar to regular expressions, but much simpler. If RELATIVE flag is specified for an expression, the results will be returned as a relative path to the given path. (We do not recommend using GLOB to collect a list of source files from your source tree. If no CMakeLists.txt file changes when a source is added or removed then the generated build system cannot know when to ask CMake to regenerate.)

You can use an easy way in this PR, but we have to find more appropriate way to do it in future.